spec/jobs/dashboard_stats_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ::Jobs::DashboardStats do
+  let(:group_message) { GroupMessage.new(Group[:admins].name, :dashboard_problems, limit_once_per: 7.days.to_i) }
+
+  def clear_recently_sent!
+    Discourse.redis.del(group_message.sent_recently_key)
+  end
+
+  before do
+    clear_recently_sent!
+  end
```

Comment:
The fact that we're clearing in a `before` block means we're leaking state in the tests here. It'll be better if we clean up after each test is done.